pkg/api/response/response.go

```diff
@@ -189,13 +203,19 @@ func Error(status int, message string, err error) *NormalResponse {
 		data["message"] = message
 	}
 
-	if err != nil {
-		if setting.Env != setting.Prod {
-			data["error"] = err.Error()
+	var overrideError *OverrideError
+	if errors.As(err, &overrideError) {
```

Comment:
One draw back in this is that every time an error is wrapped with `fmt.Errorf("some additional info: %w", err)` you have to use `%w` because you may not know if the wrapping will be significant, otherwise some middle layer could mask the error. Not sure if there is a better way though.